02-express-tutorial/controllers/people.js

```diff
@@ -0,0 +1,94 @@
+const db = require("./services");
```

Comment:
optional, nitpicky:
It is general best practice to align the name of the variable with the name of the file you are importing.  In this case, it can be confusing that `db` is coming from a files called "services."

In practice, some people will put database-specific calls into a "db" module, and then build services on top of that db abstraction layer.  There may also be intermediate layers, such as "query".